ee/cis/macos-14/README.md

```diff
@@ -0,0 +1,37 @@
+# macOS 14.0 Sonoma benchmark
+
+Fleet's policies have been written against vTODO of the benchmark. You can refer to the [CIS website](https://www.cisecurity.org/cis-benchmarks) for full details about this version.
+
+These policies cover TODO of TODO CIS items that are able to be automated. For requirements and usage details, see the [CIS Benchmarks](https://fleetdm.com/docs/using-fleet/cis-benchmarks) documentation.
```

Comment:
I'm not sure how many CIS Benchmarks Fleet covers for macOS 14. 

Hey @sharon-fdm do you know?

Reply:
macos 13 [1.0.0](https://drive.google.com/file/d/1VL8e2wmApkayv-9gM8zlyYljP6phsQhO/view?usp=share_link)
macos 14[1.0.0](https://drive.google.com/file/d/10hSWBZ06_XTwxC0GNjsjwObcQk72NQi8/view)
Win10 [2.0.0](https://drive.google.com/file/d/1agln3JBRkwJcefHibXzLMai31UuvTsDm/view)
Win11 [2.0.0](https://drive.google.com/file/d/1Ol_TKqB96M4R4QCG7vkwCbWVV6fpZZnK/view?usp=drive_link)

Reply:
How many for macOS 14? 

> These policies cover TODO of TODO CIS items that are able to be automated.

These are the TODOs in the macOS 14 CIS Benchmark docs @rachaelshaw and I are trying to wrap up.

Reply:
Sorry @noahtalerman I misled you.
There are 109 policies for macos14.

(I previously counted it wrong